tests/test_status.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import io
+import os
+import sys
+
+import generate_status_reference_data as gen
+import signac
+
+
+def test_print_status():
```

Comment:
I think that using fixtures to make this a test that is parametrized by the status arguments used would be cleaner, but this should hold the PR up: it would just make it cleaner.